ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -50,13 +50,14 @@ namespace ngraph
                         auto splits = builder::opset1::split(shape, rank_size);
                         auto num_groups_const =
                             default_opset::Constant::create(element::i64, Shape{1}, {num_groups});
-                        NodeVector new_shape{
-                            splits[0].get_node_shared_ptr(),
+                        ngraph::OutputVector new_shape{
+                            splits[0],
```

Comment:
Sorry, I've removed my previous comment by mistake.
My suggestion was to use `OutputVector` instead of `NodeVector` and as I can see it fixed the issue with split outputs:)